### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by faenir**

### CODE REVIEW

File: src/User/resources/i18n/ru/usuario.php [hunk 1]
```diff
@@ -185,9 +185,9 @@
...
     'Your account has been created and a message with further instructions has been sent to your email' => 'Аккаунт создан и дальнейшие инструкции высланы на вашу электронную почту',
-    'Your account on {0} has been created' => '',
-    'Your confirmation token is invalid or expired' => 'Токен подтверждения невалиден или истёк срок его действия',
-    '{0, date, MMMM dd, YYYY HH:mm}' => '',
+    'Your account on {0} has been created' => 'Ваш аккаунт на сайте "{0}" был успешно создан',
+    'Your confirmation token is invalid or expired' => 'Ваша ссылка устарела или является ошибочной',
+    '{0, date, MMMM dd, YYYY HH:mm}' => '{0, date, MMMM dd, YYYY HH:mm}',
     'Class "{0}" does not exist' => '@@@@',
     'Registration ip' => '@@@@',
     'Rule class can not be instantiated' => '@@@@',
```
Comment (tonydspaniard) [inline src/User/resources/i18n/ru/usuario.php]: @faenir no translations here? only three missing... 😃

**Comment by tonydspaniard** [comment]
@faenir please, add your name on the change log (in-progress)

File: src/User/resources/i18n/ru/usuario.php (hunk 1 above)
Comment (faenir) [inline src/User/resources/i18n/ru/usuario.php]: [Is it used in the source code?](https://github.com/yiisoft/yii2/blob/master/framework/console/controllers/MessageController.php#L84)

**Comment by faenir** [comment]
Done :)

**Pull request merged**